backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt

```diff
@@ -54,6 +55,16 @@ class AdministratorEntity(id: EntityID<Int>) : IntEntity(id) {
     var notificationOnApplication by Administrators.notificationOnApplication
     var notificationOnVerification by Administrators.notificationOnVerification
     var deleted by Administrators.deleted
+
+    fun isProject(project: String): Boolean =
+        this.projectId == ProjectEntity.find { Projects.project eq project }.single().id
+
+    fun isProject(projectId: Int): Boolean = this.projectId.value == projectId
+
+    fun isRole(role: Role, vararg other: Role): Boolean =
+        this.role == role.db_value || other.any { this.role == it.db_value }
```

Comment:
@wouldn't `hasRole` make more sense?
Similarly, I'd prefer `isInProject` and `isInRegion`?

Or maybe `roleIs`, `projectIs` and `regionIs`?